specs/lib/triBoxOverlapSpec.js

```diff
@@ -0,0 +1,70 @@
+'use strict';
+var Cesium = require('cesium');
+var Cartesian3 = Cesium.Cartesian3;
+var triBoxOverlap = require('../../lib/triBoxOverlap');
+
+describe('triBoxOverlap', function() {
```

Comment:
Make sure that code coverage is 100% even if the cyclomatic complexity is not.  Also test representative, but perhaps not exhaustive cases, e.g.,
* Completely inside box
* Completely outside box in each of the possible 26 voxels (or less)
* Intersecting parallel with the x plane, y plane, and z plane
* Diangle intersects
* Etc

We want to be very confident in this code so when we build on it, we don't have to chase down bugs in a more complex system.